### DISCUSSION TITLE
How do I specify I don't want all files in a subdirectory of a directory that is in sync_list?

### DISCUSSION THREAD

**Discussion by jnegrete2005** [abraunegg/onedrive#2902] [Q&A] [answered]
I have a weird question. My sync_list is like this:
```
# Exclude all venv dirs
!*venv/

# Include only linux dir and config files
/linux/
/linux/.client_config/sync_list
/linux/.client_config/config
```

I know that this sync_list file will work, but it will sync all contents of /linux/.client_config. What I want to do is only sync those two files. How should I write my sync_list file in order to accomplish this?

**Comment by abraunegg** [comment, marked as answer]
@jnegrete2005 
Please *read* the documentation: https://github.com/abraunegg/onedrive/blob/master/docs/usage.md#performing-a-selective-synchronisation-via-sync_list-file

![image](https://github.com/user-attachments/assets/82df56be-172f-41af-9869-68eb598c5026)

The way 'sync_list' operates is that it *excludes* everything by *default* and only *includes* if specifically included.

Use `--dry-run` to validate your changes.
